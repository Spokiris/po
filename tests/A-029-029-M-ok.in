5
2
33
0
0
