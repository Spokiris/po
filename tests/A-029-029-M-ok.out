Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 5
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: 2
Função a procurar: 33
Menu de Consultas
1 - Procurar valores
2 - Procurar funções
0 - Sair
Escolha uma opção: 0
Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 0
